src/tests/ftest/util/dmg_utils.py

```diff
@@ -236,15 +236,16 @@ def storage_reset(hosts, user=None, hugepages="4096", insecure=True):
     return result
 
 
-def pool_create(path, host_port, scm_size, insecure=True, group=None,
+def pool_create(path, scm_size, host_port=None, insecure=True, group=None,
```

Comment:
Do we not have any other files that call this function?  I would expect to also see changes where this method is called do to the argument order change.

Reply:
No. `create_dmg` is the only place that uses this function and I have updated it.

Reply:
Ok, good.